mandatory/3-playing-computer.js

```diff
@@ -6,37 +6,36 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). b has not been declared
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? the value of either f1 or f2 depending on the value of i
+  4. How many times is "f1" called? 2
+  5. How many times is "f2" called? 3
+  6. What value does the "a" parameter take in the first "f1" call? 1
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? 8
 */
 
 let x = 2;
 let a = 6;
 
-const f1 = function (a, b) {
-  return a + b;
+const f1 = function(a, b) {
+    return a + b;
 };
 
-const f2 = function (a, b) {
-  return a + b + x;
+const f2 = function(a, b) {
+    return a + b + x;
 };
 
 console.log(x);
 console.log(a);
-console.log(b);
 
 for (let i = 0; i < 5; ++i) {
-  a = a + 1;
-  if (i % 2 === 0) {
-    const d = f2(i, x);
-    console.log(d);
-  } else {
-    const e = f1(i, a);
-    console.log(e);
-  }
-}
+    a = a + 1;
+    if (i % 2 === 0) {
+        const d = f2(i, x);
+        console.log(d);
+    } else {
+        const e = f1(i, a);
+        console.log(e);
+    }
+}
```

Comment:
Your code is so nice and neat, very pleasant to look at.

Reply:
Thank you 😊